osmosis/assetlist.json

```diff
@@ -1,1499 +1,1525 @@
 {
-  "$schema": "../assetlist.schema.json",
-  "chain_name": "osmosis",
-  "assets": [
-    {
-      "description": "The native token of Osmosis",
-      "denom_units": [
-        {
-          "denom": "uosmo",
-          "exponent": 0
-        },
-        {
-          "denom": "osmo",
-          "exponent": 6
-        }
-      ],
-      "base": "uosmo",
-      "name": "Osmosis",
-      "display": "osmo",
-      "symbol": "OSMO",
-      "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.png",
-        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.svg"
-      },
-      "coingecko_id": "osmosis",
-      "keywords": [
-        "dex",
-        "staking"
-      ],
-      "images": [
-        {
+    "$schema": "../assetlist.schema.json",
```

Comment:
Please reset the formatting to 2-space indentation instead of the 4 you're using for the first indent

Reply:
I'm sorry i don't get this one :/ the format is exactly like the others or am i missing someting.